pkg/stack_mgr/stack_service.go

```diff
@@ -163,7 +159,7 @@ func (s *StackService) Remove(ctx context.Context, stackName string) error {
 	if err != nil {
 		return errors.Wrap(err, "unable to resync the workspace")
 	}
-	delete(stacks, stackName)
```

Comment:
local `stacks` goes away after this but we probably want to keep the cache in sync